scripts/release/create-release-metadata-for-ghcup.sh

```diff
@@ -1,114 +1,95 @@
-#!/usr/bin/env bash
+#!/bin/sh
 
-# This script, when passed the cabal release number as the first and only argument
-# generates the metadata in the correct format to be useable as is by GHCup
-# for eg:-
-# $ create-release-metadata-for-ghcup.sh 3.10.2.0 or "3.10.2.0"
-
-# Note:- Please run ./download-cabal-install-release-binaries.sh before running this script.
```

Comment:
There is a bunch of comments that were removed, and nothing seems to replace them. Could you update these top-of-the-script comments instead of removing them? Here and elsewhere.

Reply:
We have a bit of disagreement here, but I get your point, thank you.